### DISCUSSION TITLE
script com erro xtream ui mod para ubuntu 20

### DISCUSSION THREAD

**Discussion by rtmtip** [community/community#57055] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

segue o codigo com erro no mariaDB tentei instalar mais sempre da um erro

sudo apt update && sudo apt full-upgrade -y;
wget https://raw.githubusercontent.com/NeySlim/xtreamui_mirror/master/install.py;
sudo python install.py

**Comment by rtmtip** [comment]
possivel me ajudar concertando o instalador pois esse mod éra muito bom!

**Reply by MINmix1978** [reply]
O mesmo erro aqui, já tentei de tudo e nao instala LB

**Comment by rtmtip** [comment]
o erro esta nessa linha !  http://mirror.lstn.net/mariadb/repo/10.5/ubuntu

**Reply by MINmix1978** [reply, marked as answer]
Muda esse link para este! 

http://ams2.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu

Resolve o problema.

**Comment by otter-computer** [comment]
Hi @rtmtip, thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
